.appveyor.yml

```diff
@@ -0,0 +1,19 @@
+image: ubuntu1804
+version: '{branch}-{build}'
+
+configuration:
+  - build.APMrover2 drive.APMrover2
+  - build.ArduCopter fly.ArduCopter
```

Comment:
We already test this elsewhere.  If you want to keep things short you could remove this relatively long-running test.